cmake/projects/FP16/hunter.cmake

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2016-2019, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+hunter_add_version(
+    PACKAGE_NAME
+    FP16
+    VERSION
+    0.0.0-febbb1c-p1
+    URL
+    "https://github.com/hunter-packages/FP16/archive/v0.0.0-febbb1c-p0.tar.gz"
```

Comment:
> https://travis-ci.org/xsacha/hunter/jobs/507674728

> -- Using src='https://github.com/xsacha/FP16/archive/v0.0.0-febbb1c-p1.tar.gz'

You're testing and submitting different versions...

Reply:
Updated tests. It was same code